team-validator.js

```diff
@@ -214,6 +215,14 @@ class Validator {
 				return [`"${set.ability}" is an invalid ability.`];
 			}
 		}
+		if (tools.getNature(set.nature).id && !tools.getNature(set.nature).exists) {
```

Comment:
Taking that out triggers two build errors: ``1) Team Validator features TeamValidator should accept legal movesets`` &  ``2) Team Validator features TeamValidator should accept both ability types for Mega Evolutions``